test/core/runtime/wasm_executor_test.cpp

```diff
@@ -41,27 +60,12 @@ class WasmExecutorTest : public ::testing::Test {
  * @then proper result is returned
  */
 TEST_F(WasmExecutorTest, ExecuteCode) {
-  auto res =
-      executor_->call(wasm_provider_->getStateCode(),
-                      external_interface_,
-                      "addTwo",
-                      wasm::LiteralList{wasm::Literal(1), wasm::Literal(2)});
-  ASSERT_TRUE(res) << res.error().message();
-  ASSERT_EQ(res.value().geti32(), 3);
-}
+  EXPECT_OUTCOME_TRUE(module_and_memory, runtime_manager_->getModuleInstance());
+  auto &&[module, memory] = std::move(module_and_memory);
 
-/**
- * @given wasm executor
- * @when call is invoked with invalid or empty state code
- * @then proper error is returned
- */
-TEST_F(WasmExecutorTest, ExecuteWithInvalidStateCode) {
```

Comment:
What happened with this test?

Reply:
It was test for situation which impossible any more. Invalid state code will catched near before.